src/extension.ts

```diff
@@ -1575,17 +1577,17 @@ export async function openPS(node: IZoweDatasetTreeNode, previewMember: boolean,
     if (validProfile === 0) {
         try {
             let label: string;
-            switch (node.getParent().contextValue) {
-                case (FAVORITE_CONTEXT):
+            switch (true) {
```

Comment:
Could we change this to regular `if...else` logic?
I don't know if `switch(true)` would be confusing for others reading this, but it confused me for a second 😋

Reply:
Seemed to be a recognised approach but am happy to change it if preferred